semgrep_metrics.atd

```diff
@@ -9,9 +9,19 @@
  * translated originally from semgrep/cli/.../metrics.py
  *
  * TODO:
- * - some optional arguments should be removed when `osemgrep` replace
- *   `semgrep`. Currently, we tried to fit on tests and don't update the
- *   JSON output. We tried to fit on the Python side too.
+ * - some optional arguments could be removed when `osemgrep` replace
+ *   `pysemgrep`. Currently, we try to fit the tests and don't update the
+ *   JSON output.
+ * - specify the schema for the answer from https://metrics.semgrep.dev
+ *   Here is an example of answer:
```

Comment:
@amchiclet I can reproduce those errors with
running `./bin/osemgrep --experimental --config semgrep.jsonnet --error --strict --exclude tests --debug` in the semgrep repo (essentially what `make check` is running).
Looks like osemgrep can't even send metrics currently ...